examples/covariance/plot_outlier_detection.py

```diff
@@ -63,21 +65,23 @@
         contamination=outliers_fraction)}
 
 # Compare given classifiers under given settings
-xx, yy = np.meshgrid(np.linspace(-7, 7, 100), np.linspace(-7, 7, 100))
-n_inliers = int((1. - outliers_fraction) * n_samples)
+xx, yy = np.meshgrid(np.linspace(-7, 7, n_samples // 2),
+                     np.linspace(-7, 7, n_samples // 2))
```

Comment:
It's not clear to me why the precision of the grid should be related to the number of samples.

Reply:
Yes, thanks - that was just my being confused. I've now introduced a `GRID_PRECISION` constant instead.